### DISCUSSION TITLE
Change default user

### DISCUSSION THREAD

**Discussion by iamjonasmarlo** [coder/code-server#3623] [Q&A]
I'm using code-server on a digital ocean droplet with nginx. Is it possible to change the default user in the integrated terminal? Currently it's the root user.

Thank you!

**Comment by jsjoeio** [comment]
Hmm...I think this has been asked before but I can't remember the exact solution. Possibly related:
- https://github.com/cdr/code-server/issues/3202
- https://www.digitalocean.com/community/questions/how-to-change-the-username-root
- https://github.com/cdr/code-server/discussions/3055#discussioncomment-577942

Let me know if that resolves your issue!
